README.md

```diff
@@ -38,6 +38,10 @@ Ceph contains many git submodules that need to be checked out with
 
 ## Build Prerequisites
 
+A debug build of Ceph with the source repo takes around 40 gigabytes. If you want to build Ceph in a virtual machine (VM) please make sure total disk space on the VM is at least 60 gigabytes.
+
+Please also be aware that some distributions of Linux, like CentOS, use Linux Volume Manager (LVM) for the default installation. LVM may reserve a large portion of disk space of a typical sized virtual disk for the operating system.
```

Comment:
i'd avoid putting more detailed notes or instructions into `README.md`. as personally, i think the longer a document is, the less chance people will read through it.

Reply:
Removed from README.md

Kept in doc/install/build-ceph.rst